lib/toolbar-android.js

```diff
@@ -29,53 +29,86 @@ export default function createToolbarAndroidComponent(IconNamePropType, getImage
 
     static defaultProps = {
       iconSize: 24,
+      actions: [],
     };
 
     updateIconSources(props) {
```

Comment:
This method is really messy now. I'd like for it to change to be `getNextState` or something that will return a promise that will return a promise resolving to an object with `{ navIcon, actions, etc... }`. Any empty values should be `undefined` so we don't have to evict them from state explicitly.